source/style-guide/information-types/prototypes/reference.txt

```diff
@@ -27,6 +27,31 @@ contents, often called a short description. The short description
 explains a little about how the reference material is used, and may link
 to a concept or task page.
 
+Compatibility 
+-------------
+
+Optional. 
+
+A list of MongoDB deployment types that support the referenced method, operator, 
```

Comment:
```suggestion
A list of MongoDB editions that support the referenced method, operator, 
```
Suggestion/Question: Confused by the term "deployment type", is that the term we want to use? The Enterprise Advanced version and Community version of MongoDB are described as 'editions' on their landing page or title, respectively. Perhaps we could use 'edition' instead.

Reply:
Noted and updating!